x/lightclient/types/block_header_verification.go

```diff
@@ -0,0 +1,82 @@
+package types
```

Comment:
I think we can also add a validate method that check no duplicated chain IDs and all chain IDs are valid. And add this validate to genesis validate

We can also create an issue and add it to the block header epic since this module is not a priority for now